api/src/templates/updated_member_info.subject.html

```diff
@@ -0,0 +1 @@
+Personal information updated
```

Comment:
```suggestion
Stockholm Makerspace: Personal information updated
```

Reply:
The sender's name is "Stockholm Makerspace", so adding this would be duplicate info

Reply:
Then I keep this subject text